skimage/segmentation/_watershed.py

```diff
@@ -92,32 +92,36 @@ def watershed(
     compactness=0,
     watershed_line=False,
 ):
-    """Find watershed basins in `image` flooded from given `markers`.
+    """Find watershed basins in an image flooded from given markers.
 
     Parameters
     ----------
     image : ndarray (2-D, 3-D, ...)
         Data array where the lowest value points are labeled first.
-    markers : int, or ndarray of int, same shape as `image`, optional
-        The desired number of markers, or an array marking the basins with the
+    markers : int, or ndarray of int with same shape as image, optional
+        The desired number of basins, or an array marking the basins with the
         values to be assigned in the label matrix. Zero means not a marker. If
-        ``None`` (no markers given), the local minima of the image are used as
-        markers.
+        None, the (default) markers are determined as the local minima of
+        ``image``. Specifically, the computation is equivalent to applying
+        :func:`skimage.morphology.local_minima` onto ``image``, followed by
+        :func:`skimage.measure.label` onto the result (with the same given
+        ``connectivity``). Generally speaking, users are encouraged to pass
+        markers explicitly.
     connectivity : ndarray, optional
-        An array with the same number of dimensions as `image` whose
+        An array with the same number of dimensions as ``image`` whose
```

Comment:
I see: "Enclose variables in single backticks." I was convinced of the opposite, because single backticks render just italic, whereas double backticks render like this:
![double_backticks](https://github.com/scikit-image/scikit-image/assets/2227806/57e3c81b-c92d-4208-a7db-5a8b75a20781)
Then I must update this elsewhere as well; and our documentation is full of variables enclosed in double backticks... :grimacing:

Reply:
Using single backticks in rST corresponds to emphasis rather than italics. Italics just happens to be what it is formatted as. At least that is how I explain that convention to myself. 

> and our documentation is full of variables enclosed in double backticks... 😬

As with many other inconsistencies. :sweat_smile:

Reply:
> emphasis rather than italics. Italics just happens to be what it is formatted as.

Yes, yes,, I know. I wrote the comment quickly and I actually thought to myself "I should be *semantic* and write 'emphasis' but, oh, well" :sweat_smile: But I find this consideration unrelated:

> At least that is how I explain that convention to myself.

What does it explain? It's not intuitive to me at all that *variables* would be, by (any) convention, represented with emphasis style. To me, that 'steals' the emphasis use away from its original/natural/intuitive purpose: I would emphasize a *word* in a (plain English) sentence, in a docstring just like I would in any text. Now, if *emphasis* is used (by convention) to mean 'variable...' what can I use to emphasize a word or a group of words in a sentence? Conversely, I find that the label-like rendering of the double backticks is quite suitable/intuitive to mean 'this is a variable, not just any word in the sentence.'

> As with many other inconsistencies. 😅

For sure...